.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ '*' ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      OCTOPUS_VERSION: 4.3.${{ github.run_number }}
```

Comment:
For anyone else who is looking, this version is done like this on purpose, it's a temporary measure while we're moving the builds and then updating our versioning.

Reply:
💯